mods/other/speed_nodes/init.lua

```diff
@@ -0,0 +1,94 @@
+minetest.register_node("speed_nodes:cobweb", {
```

Comment:
```suggestion
local players = {}
minetest.register_node("speed_nodes:cobweb", {
```
IIRC the mod won't work without this, also luacheck hates you because of this missing

Reply:
"players" is from the physics mod

Reply:
Is it global though?

Reply:
Yes, I globalized it.